scikeras/__init__.py

```diff
@@ -8,8 +8,7 @@
 # See https://github.com/tensorflow/tensorflow/pull/42097
 # Will be removed whenever the
 # min supported version of tf incorporates the fix
-import tensorflow.python.keras.metrics  # noqa
```

Comment:
This import is broken in `tf-nightly` as of 20200810. This change fixes that, at least for now.